en/asset/model/glTF.md

```diff
@@ -144,20 +145,20 @@ glTF 贴图缩小筛选器将按照下表中的映射关系转换为 Cocos Creat
 | `NEAREST_MIPMAP_LINEAR`  | `TextureBase.Filter.NEAREST` | `TextureBase.Filter.LINEAR`       |
 | `NEAREST_MIPMAP_NEAREST` | `TextureBase.Filter.LINEAR`  | `TextureBase.Filter.LINEAR`       |
 
-glTF 贴图环绕模式将按照下表中的映射关系转换为 Cocos Creator 贴图环绕模式：
+glTF Texture Wrap mode will be converted to Cocos Creator Texture Wrap mode according to the texture in the following table:
 
-| glTF 贴图环绕模式   |       Cocos Creator 贴图环绕模式         |
+| glTF Texture Wrap Mode | Cocos Creator Texture Wrap Mode |
 | :---------------- | :------------------------------------- |
 | `CLAMP_TO_EDGE`   | `TextureBase.WrapMode.CLAMP_TO_EDGE`   |
 | `REPEAT`          | `TextureBase.WrapMode.REPEAT`          |
 | `MIRRORED_REPEAT` | `TextureBase.WrapMode.MIRRORED_REPEAT` |
 
-### glTF 图像
+### glTF Image
 
-导入后，glTF 图像将转换为 Cocos Creator 中的图像资源。
+After import, the glTF image will be converted to an image asset in Cocos Creator.
 
-当 glTF 图像的 [URI](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#imageuri) 是 Data URI 时，图像数据将从 Data URI 中获取。否则，将根据 [Cocos Creator 图像位置解析算法](./image-location-resolution.md) 解析并引用外部图像文件，其中 `url` 就是 glTF 图像的 URI，`startDir` 为 glTF 文件所在目录。
+When the [URI](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#imageuri) of the glTF image is a Data URI, the image data will be fetched from the Data URI. Otherwise, the image data will be retrieved from the Data URI according to the [Cocos Creator Image Location Resolution Algorithm](./image-location-resolution.md), where `url` is the URI of the glTF image and `startDir` is the directory where the glTF file is located.

```

Comment:
```suggestion
When the [URI](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#imageuri) of the glTF image is a Data URI, the image data will be fetched from the Data URI. Otherwise, the image data will be resolved from the Data URI according to the [Cocos Creator Image Location Resolution Algorithm](./image-location-resolution.md), where `url` is the URI of the glTF image and `startDir` is the directory where the glTF file is located.

```